src/components/atoms/ChatMessage/ChatMessage.scss

```
@@ -96,7 +96,9 @@ $reply-icon-radius: 20px;
   }
 
   &__reply-icon {
-    display: none;
+    color: white;
```

Why do we need to explicitly set this color here anyway?

```suggestion
    color: $white;
```